the-collab-lab/tcl-76-smart-shopping-list
# add general styles 23

## Description

- Imported Tailwind.CSS
- Imported DaisyUI
- Imported Archivo Narrow font
- Added Lemonade and Coffee as themes in the `tailwind.config` file
- Added `font-archivo` in `home.jsx` as a preview 

## Related Issue

closes #42 (#23)

## Acceptance Criteria

- [x] Colors and fonts are established and are consistent across pages
- [ ] There is a dark mode toggle that a user can activate to change the display of our app


## Type of Changes

`enhancement`

## Updates

### After

<img width="936" alt="Screenshot 2024-09-30 at 12 42 47 PM" src="https://github.com/user-attachments/assets/8963383c-b264-4a65-9adb-e03f8edaee16">

## Testing Steps / QA Criteria

- install new packages with `npm i` / `npm install `
- Should be able to add Archivo font as `font-archivo` as a class for specific elements as see the font difference. Right now it is only in `Home.jsx` added as a class on the `ul` for the list of lists.
- Should see a lemon yellow background on all pages
